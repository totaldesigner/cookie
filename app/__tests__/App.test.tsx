import * as React from 'react'
import App from '../App'

import * as renderer from 'react-test-renderer'

describe('Testing App component', () => {
  it('renders without crashing', () => {
    const rendered = renderer.create(<App />).toJSON()
    expect(rendered).toMatchSnapshot()
    expect(rendered).toBeTruthy()
  })
})