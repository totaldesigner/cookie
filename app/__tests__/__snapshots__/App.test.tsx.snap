// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App component renders without crashing 1`] = `<ExponentAppLoading />`;
